src/Microsoft.XmlSerializer.Generator/pkg/build/Microsoft.XmlSerializer.Generator.targets

```diff
@@ -17,11 +17,44 @@
     <Warning Condition="Exists('$(SerializerDllImmediatePath)') != 'true' And Exists('$(SerializerCsImmediatePath)') == 'true'" Text="$(SGenWarningText)"/>
     <Copy Condition="Exists('$(SerializerDllImmediatePath)') == 'true'" SourceFiles="$(SerializerDllImmediatePath)" DestinationFolder="$(OutputPath)" />
   </Target>
+  
   <Target Name="CleanSerializationAssembly" AfterTargets="CoreClean">
     <Message Text="Cleaning serialization files..." Importance="normal"/>
     <Delete Condition="Exists('$(OutputPath)\$(SerializerName).dll') == 'true'" Files="$(OutputPath)\$(SerializerName).dll" />
   </Target>
+  
   <Target Name="CopySerializer" AfterTargets="PrepareForPublish">
```

Comment:
>CopySerializer [](start = 16, length = 14)

To keep consistent naming with the other targets, this should be CopySerializationAssembly